### PR TITLE
Finedesign share panel

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Works towards https://github.com/owncloud/web/issues/7340

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal2</strong> https://drone.owncloud.com/owncloud/web/27175/61/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsers-shareWithUsers_feature-L139.png
  

### webUISharingInternalUsers-shareWithUsers_feature-L322.png
  

### webUISharingInternalUsers-shareWithUsers_feature-L331.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers_feature-L34.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers_feature-L35.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers_feature-L36.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers_feature-L37.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by kulmann — CHANGES_REQUESTED** [review]
I see quite some code deletion in this PR that don't have clear acceptance criteria in the linked issue. Would like to clarify with @tbsbdr here or in the issue if e.g. removing the file size info or invite help text is really intended or just missing in the design.

**Comment by kulmann** [comment]
Please note that I changed the PR description to state `Works towards #7340` instead of fixing it, because rearranging the dropdowns will happen in another PR https://github.com/owncloud/web/pull/7365

**Comment by JanAckermann** [comment]
@kulmann This has been spoken through in detail, all code deletion was done with intention

**Comment by kulmann** [comment]
> @kulmann This has been spoken through in detail, all code deletion was done with intention

Then let's please take the time and write down the decisions in the issue

**Comment by kulmann** [comment]
Needs a rebase now

**Comment by JanAckermann** [comment]
done

File: packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue [hunk 1]
```diff
@@ -149,34 +141,6 @@ export default defineComponent({
...
-    spaceAddMemberHelp() {
-      return shareSpaceAddMemberHelp
-    },
-
-    inviteDescriptionMessage() {
-      if (this.capabilities.files_sharing.federation?.outgoing === true) {
-        return this.$gettext('Add new person by name, email or federation IDs')
-      }
-      const cernFeatures = !!this.configuration?.options?.cernFeatures
-      return cernFeatures
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue:175]: @elizavetaRa @diocas decision was made to remove the description message below the sharing invite form. If you can't drop the line from your UI we'll have to find a different solution. Please reach out then.

**Review by kulmann — APPROVED** [review]
Cool 👍

**Pull request merged**